server/config/config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "bolaji",
+    "database": "hello-books",
+    "host": "127.0.0.1",
+    "port": 5432,
+    "dialect": "postgres",
+    "operatorsAliases":  {
+      "$and": "Op.and",
+      "$or": "Op.or",
+      "$eq": "Op.eq",
+      "$gt": "Op.gt",
+     " $lt": "Op.lt",
+      "$lte": "Op.lte",
+      "$like": "Op.like"
+    }
+  },
+  "test": {
+    "username": "postgres",
+    "password": "bolaji",
+    "database": "hello-bookTest",
+    "host": "127.0.0.1",
+    "port": 5432,
+    "dialect": "postgres"
+  }
+}
```

Comment:
I should not be seeing the password and other details to your server here. You should place this under an environmental variable file named `env`, which you must not push to Github. You should include it under your `gitignore` file  so that Git does not place it under un-staged files.

Reply:
Okay. I will do as you have said.